src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,49 +1,101 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int ARRAY_SIZE = 10;
+    private Object[] array;
+    private int counter = 0;
+
+    public ArrayList() {
+        array = new Object[ARRAY_SIZE];
+    }
+
+    private void resize(int minSize) {
```

Comment:
let’s move private methods to the bottom of the file, or at least after the place where we call them